src/actions/index.js

```diff
@@ -0,0 +1,4 @@
+export const addSighting = eventDetails => ({
```

Comment:
This action looks good, once we have a backend we should be able to start testing it